hbase-server/src/main/java/org/apache/hadoop/hbase/wal/AbstractWALRoller.java

```diff
@@ -58,31 +58,31 @@
 
   protected static final String WAL_ROLL_PERIOD_KEY = "hbase.regionserver.logroll.period";
 
-  protected final ConcurrentMap<WAL, Boolean> walNeedsRoll = new ConcurrentHashMap<>();
+  protected final ConcurrentMap<WAL, RollController> wals = new ConcurrentHashMap<>();
```

Comment:
walRolls? instead of wals?

Reply:
rollWals? more suitable？

Reply:
This will have all wal entries once the wal instance is created.  Not just when it needs a roll. So ya these were wals which needed roll at some point.  So wals name also fine IMO.

Reply:
Ok